scopes/lanes/lanes/switch.cmd.ts

```diff
@@ -5,33 +5,33 @@ import { MergeStrategy } from '@teambit/legacy/dist/consumer/versions-ops/merge-
 import { LanesMain } from './lanes.main.runtime';
 
 export class SwitchCmd implements Command {
-  name = 'switch <lane> [pattern]';
+  name = 'switch <lane> [pattern]'; // @david why do we have the pattern flag here?
   description = `switch to the specified lane`;
   private = true;
   alias = '';
   arguments = [
     {
       name: 'lane',
-      description: 'lane-name or lane-id (if not exists locally) to switch to',
+      description: 'lane-name or lane-id (if lane not imported/created locally) to switch to',
```

Comment:
i don't like the "imported/created". there is no difference between the two. it's just a lane that is available locally

Reply:
I prefer 'if lane is not local' then, but we need to make sure our docs or somewhere explains what a local lane is (and isn't)

(If it wasnt locally available then how can i locally run the command and switch to the lane?)

Reply:
i am not aware how this is explained in the docs. there were several recent updates to lanes-docs. so we can ensure this is in-sync